tests/testthat/test_calc.dtsurvey.R

```diff
@@ -410,3 +411,93 @@ test_that('Where elimiates everything', {
 
 })
 
+test_that('MI approach', {
+
+  midat = lapply(1:10, function(x){
+    r = apiclus1
+    r$random = sample(1:5, nrow(r), T)
+    r$random3 = sample(1:3, nrow(r), T)
+    r$random_fact = factor(r$random)
+    dtsurvey(r, 'dnum', weight = 'pw')
+
+  })
+
+  midat = mitools::imputationList(midat)
+  misur = svydesign(id=~dnum, weights=~pw, data=midat)
+
+  # Numeric variable unrelated
+  r1.1 = mitools::MIcombine(with(misur, svymean(~api00,design = misur)))
+  r1.2 = calc(midat, 'api00', metrics = c('mean', 'vcov'))
+  r1.3 = calc(midat$imputations[[1]], 'api00')
```

Comment:
Consider adding the following test for good measure?
`expect_equal(r1.2$mean_se, r1.3$mean_se)`